catalogue/webapp/components/ExpandedImage/ExpandedImage.js

```diff
@@ -72,12 +79,35 @@ const LicenseWrapper = styled.div`
 `;
 
 const Content = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
   padding: 10px 0 0 0;
   ${({ theme }) => theme.media.large`
     padding: 0 0 0 30px;
+    max-width: 50%;
   `}
 `;
 
+const SpacedButton = styled(Button)`
+  margin: 10px 10px 0 0;
+`;
+
+// The padding combined with the 2px border on a `secondary` button means
+// that it appears larger than the `primary` button despite border-box being set.
+// This styling - reducing the padding by the border width - makes the buttons the same size
+const SpacedButtonBorderBox = styled(SpacedButton)`
+  padding: ${({ theme: { spacingUnit } }) =>
+    `${spacingUnit - 2}px ${4 * spacingUnit - 2}px`};
+`;
+
+const RelatedImagesWrapper = styled.div`
+  display: flex;
+  flex-grow: 10;
```

Comment:
What's the upshot of this? `flex-grow` never does what I _think_ it should do!

Reply:
I wanted the related images pinned to the bottom of the panel but didn't want them absolutely positioned as that would cause overlaps: setting `flex-grow` to an (arbitrary) number greater than 1 here makes sure that this div takes up the maximum height that it can, and so the `end` in `justify-end` is the bottom of the panel .

Reply:
> flex-grow never does what I think it should do!
Same - this took me a while to work out haha